lib/jwe/encrypt.js

```diff
@@ -379,6 +379,18 @@ function JWEEncrypter(cfg, fields, recipients) {
         }
         props.iv = iv;
 
+        if ("recipients" in jwe && jwe.recipients.length > 0) {
```

Comment:
I'd prefer this were an "=== 1" check, since the suite of "A*CBC+H*" are really single-recipient algorithms.

Also, (as much as I'd rather it not do this) I think there ought be a reject somewhere near here if there is more than one recipient and one of the "A*CBC+HS*" algorithms is used for content encryption.  I can live without that particular check for now, though.